Mage/src/main/java/mage/abilities/keyword/BountyAbility.java

```diff
@@ -20,15 +20,16 @@ public class BountyAbility extends DiesCreatureTriggeredAbility {
     }
 
     public BountyAbility(Effect effect) {
-        super(effect, false, bountyCounterFilter);
+        this(effect, false);
     }
 
     public BountyAbility(Effect effect, boolean optional) {
-        super(effect, optional, bountyCounterFilter);
+        this(effect, optional, false);
     }
 
     public BountyAbility(Effect effect, boolean optional, boolean setTargetPointer) {
         super(effect, optional, bountyCounterFilter, setTargetPointer);
+        setTriggerPhrase("<i>Bounty</i> &mdash; ");
```

Comment:
Looking at the cards that use this ability (they're all from the fanmade Star Wars set) it seems the trigger phrase is actually supposed to be "<i>Bounty</i> &mdash; Whenever a creature an opponent controls with a bounty counter on it dies, ". All cards with this ability have broken rules text that doesn't actually say what the trigger condition is (I guess whoever made the Star Wars set didn't understand the difference between ability words and keywords...)